### DISCUSSION TITLE
Api com redundância

### DISCUSSION THREAD

**Discussion by marconneves** [BrasilAPI/BrasilAPI#180] [Ideas]
Opa pessoal, cheguei aqui nesta api maravilhosa que tanto precisava atravez do nosso Teló hehe, mas então, estava pensando em fazer uma segunda opção desta api com redundância de dados usando redis ou mongodb, como opsão de fallback, pensei aqui nas seguintes opções:
1. Fazer um fock da mesma e manter la esta segunda opção com a redudancia.
2. Criar quem sabe uma nova branch neste mesmo repo com a redudancia.

Aqui na empresa que trabalho seria muito interessante isso, e consegui até um apoio para trabalhar neste repo quando estiver meio desafogado aqui, gostaria de saber a opnião de vocês sobre o mesmo.

**Comment by marconneves** [comment]
Em relação aos DBs, o bom de usar o redis é que a resposta seria muito rápida, porem não temos opções gratuitas, ai pensei no MongoDB, que podemos conseguir uma faixa la no MongoCloud gratuito até uma quantidade de dados.

**Reply by gustavo-oliveira-rosa** [reply]
Seria legal começarmos pelo mongodb e ir começar uma vaquinha de apoio para a cota de dados excedida

**Comment by farukaf** [comment]
Tem uma issue pra colocar o banco dod CNPJ fornecidos dentro do proj. Hj é utilizado uma api de terceiro.

Creio que redis como performance seria uma redundância estranha dado a estrutura atual
